modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -29,22 +34,31 @@ spec:
   agent:
     type: eBPF
     ebpf:
+      privileged: true <1>
+      features:
+        - PacketDrop
       flowFilter:
-        action: Accept  <1>
-        cidr: 172.210.150.1/24 <2>
-        protocol: SCTP
-        direction: Ingress
-        destPortRange: 80-100
-        peerIP: 10.10.10.10
-        enable: true    <3>
+        enable: true <2>
```

Comment:
Thinking more about this, I feel instead of this example we can use the `Filter EastWest and NorthSouth flows` example in blog https://netobserv.io/posts/enhancing-netobserv-by-introducing-multi-rules-flow-filtering-capability-in-ebpf/